source/app/main.c

```diff
@@ -23,27 +24,51 @@ Audio loadAudio() {
     return loadAudioFromROM(audioData);
 }
 
+void seedRNGByKeyPress() {
+  /* fake seeding by just fetching numbers until key is pressed. */
+  while(1) {
+    vid_vsync();
+    SimpleRNG_rand();
+    key_poll();
+    if(key_hit(KEY_ANY)){
```

Comment:
```suggestion
    if (key_hit(KEY_ANY)) {
```
De spatie na `if` is een smaak, dus daar kunnen we nog over discusseren :upside_down_face:

Reply:
Ik heb hier geen voorkeur over. Typisch iets om consequent te doen, en een formatter dus goed voor is.